modules/test/playwright/tests/frontend-js-aui-web/main/menu.spec.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../../fixtures/loginTest';
+
+export const test = mergeTests(
+    loginTest(),
+);
+
+test(
+    'Check dropdown menu has show',
+    {tag: '@LPD-61908'},
+    async ({page}) => {
+        await page.getByRole('button', { name: 'Add' }).click();
+
+        await expect(page.locator('div.open.show.lfr-icon-menu-open')).toBeVisible();
```

Comment:
In the video you shown, it took some small amount of time for the menu to be closed by jquery. Here in the test, we are asserting the visibility immediately. Would this make the test pass even if your fix wasn't there?

If that's the case (have not checked this though) we'd need some reasonable delay to ensure jQuery has time to do its job

Reply:
It should be okay as it is testing to see if show is added but I went ahead and redid the test so that it is a correct representation of reproducing the issue.